.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Internal"]
+	path = Internal
+	url = git@code.is.adyen.com:iOS/adyen-checkout-ios-internal.git
```

Comment:
hm.. `.gitmodule` supposed to be ignored..

Reply:
Did I miss something on the project setup maybe? I haven't change the .gitignore file